Src/ILGPU/Runtime/OpenCL/API/CLAPI.cs

```diff
@@ -759,8 +759,10 @@ public static CLError CreateContext(IntPtr device, out IntPtr context)
             int workDimensions,
             IntPtr* workOffsets,
             IntPtr* globalWorkSizes,
-            IntPtr* localWorkSizes) =>
```

Comment:
ISSUE: Sorry, wasn't clear - you only needed to copy/paste the changes for adding the new APIs, not the changes for EnqueueBarrier, which is to fix a different bug.

Reply:
I'll fix that right away.